libs/banner/website/src/lib/banner.tsx

```diff
@@ -122,26 +141,30 @@ export const Banner = ({data, loading, error, className}: BuilderBannerProps) =>
           style={{backgroundImage: `url(${data?.primaryBanner.image.url})`}}></BannerImage>
       )}
 
-      <BannerContent>
-        <BannerTitle>{data?.primaryBanner.title}</BannerTitle>
-        <BannerText>{data?.primaryBanner.text}</BannerText>
-        {data?.primaryBanner.cta && <BannerCta>{data?.primaryBanner.cta}</BannerCta>}
-
-        <BannerActions>
-          {data?.primaryBanner.actions &&
-            data?.primaryBanner.actions.map(a => (
-              <Link
-                href={a.url}
-                key={a.url}
-                onClick={e => handleActionClick(e, a)}
-                data-role={a.role}>
-                <Button color={a.role === BannerActionRole.Primary ? 'primary' : 'secondary'}>
-                  {a.label}
-                </Button>
-              </Link>
-            ))}
-        </BannerActions>
-      </BannerContent>
+      {htmlContent != null && htmlContent != '' ? (
+        <BannerContent dangerouslySetInnerHTML={{__html: htmlContent}} />
+      ) : (
```

Comment:
If `html` is set in the editor, only show the HTML and the close button and nothing else.